### PR TITLE
Standard engine 3.x

### PR DESCRIPTION
**Pull request by wesleytodd**
This pulls in master from standard.  There was a bunch of changes, and some breaking.  So this would for sure be a major version bump.  To be honest, it would be really nice if we could track directly to standard version numbers, which would make this `6.0.7`.  But I understand if you don't want to do that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [![npm][npm-image]][npm-url]
 [![downloads][downloads-image]][downloads-url]
 
+<<<<<<< HEAD
```
Comment (tnguyen14) [inline README.md]: looks like this is an unresolved merge conflict?

Comment (wesleytodd) [inline README.md]: good catch!!! FIXED

File: RULES.md [hunk 2]
```diff
@@ -223,5 +223,118 @@ your code.
 * Always use semicolons.
 
   ```js
+<<<<<<< HEAD
```
Comment (tnguyen14) [inline RULES.md]: another merge conflict. seems pretty easy to resolve, just remove the HEAD version.

Comment (wesleytodd) [inline RULES.md]: Fixed, thanks again.  This time I did a grep for these and they should all be resolved :)

**Comment by tnguyen14** [comment]
LGTM!

**Comment by dcousens** [comment]
Nicely done @wesleytodd :)

**Comment by wesleytodd** [comment]
:) thanks!!

**Pull request merged**